model.py

```diff
@@ -153,7 +153,8 @@ def _get_completion_kwargs(self):
 
 
 supported_models: List[Model] = [
-    Model("gpt-4o-2024-05-13"),
+    Model("gpt-4o-2024-08-06"),
+    Model("gpt-4o-mini-2024-07-18"),
     Model("gpt-4-turbo-2024-04-09"),
```

Comment:
please remove the followings:
gpt-4 and gpt-4-turbo, gpt-3.5-turbo
claude 3-opus, sonnet, haiku
mistral-small, large
llama-3
eeve

add the followings:
3.5 sonnet
gemini flash
solar preview (https://huggingface.co/upstage/solar-pro-preview-instruct)

Reply:
Updated except for the solar model because they don't allow to set a specific model. solar-pro-preview-240910 is not allowed, only solar-pro is.